qcengine/programs/cfour/methods.py

```diff
@@ -0,0 +1,51 @@
+def muster_modelchem(method, derint):
+    method = method.lower()
+    opts = {}
+
+    if derint == 0:
+        if method == 'cfour':
+            pass  # permit clean operation of sandwich mode
+        else:
+            opts['deriv_level'] = 'zero'
+
+    elif derint == 1:
+        opts['deriv_level'] = 'first'
+
+    elif derint == 2:
+        opts['vibration'] = 'exact'
+
+    if method == 'cfour':
+        pass
+
+    elif method in ['scf', 'hf']:
+        opts['calc_level'] = 'scf'
+
+    elif method == 'mp2':
+        opts['calc_level'] = 'mp2'
+
+    elif method == 'mp3':
+        opts['calc_level'] = 'mp3'
+
+    elif method == 'mp4(sdq)':
+        opts['calc_level'] = 'sdq-mp4'
+
+    elif method == 'mp4':
```

Comment:
Can we have like a:
```
elif method in [...]:
   opts['calc_level'] = method
````
or do we expect these to become much more complex?

Reply:
They are/will get more complex in qcdb. Condensed might be ok here.

The `-q` is for a quiet install to prevent the log overflow.